components/vf-box/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 2.1.0
+
+- updates spacing design tokens
+
```

Comment:
On these could we add a note that it `requires vf-design-tokens@2.0.0 or newer`?

Reply:
yep - when I opened this up this morning I did think it was a little 'vague'.